crates/common/tedge_config/src/tedge_config_cli/tedge_config.rs

```diff
@@ -435,8 +435,8 @@ define_tedge_config! {
         },
 
         /// Set of MQTT topics the AWS IoT mapper should subscribe to
-        #[tedge_config(example = "tedge/measurements,tedge/measurements/+")]
-        #[tedge_config(default(value = "tedge/measurements,tedge/measurements/+,tedge/alarms/+/+,tedge/alarms/+/+/+,tedge/events/+,tedge/events/+/+,tedge/health/+,tedge/health/+/+"))]
+        #[tedge_config(example = "te/+/+/+/+/a/+,te/+/+/+/+/m/+,te/+/+/+/+/e/+")]
+        #[tedge_config(default(value = "te/device/+/+/+/m/+,te/device/+/+/+/e/+,te/device/+/+/+/a/+,tedge/health/+,tedge/health/+/+"))]
```

Comment:
The `device` prefix must not be enforced.

```suggestion
        #[tedge_config(default(value = "te/+/+/+/+/m/+,te/+/+/+/+/e/+,te/+/+/+/+/a/+,tedge/health/+,tedge/health/+/+"))]
```

Reply:
fixed